tests/test_tests___main__.py

```diff
@@ -10,127 +10,161 @@
 class ConvertArgsTests(unittest.TestCase):
 
     def test_no_args(self):
-        argv, env, runtests, lint = convert_argv([])
+        runtests, lint, runtest_args = convert_argv([])
 
-        self.assertEqual(argv, [
+        self.assertEqual(runtest_args.argv, [
             sys.executable + ' -m unittest',
```

Comment:
Not sure where this is coming from, but the `-m` and `unittest` parts should be separate list items (assuming this command will be run at some point, at which point quoting rules may corrupt the first item here)

Reply:
In this case the code is correct.  Unittest is quirky in a few spots like this.